mininet/wifiMobility.py

```diff
@@ -180,7 +180,8 @@ def associate_wpa(self, sta, ap, wlan):
         :param wlan: wlan ID
         """
         passwd = self.verifyPasswd(sta, ap, wlan)
-        os.system('pkill -f \'wpa_supplicant -B -Dnl80211 -i %s\'' % sta.params['wlan'][wlan])
+        confname = "mn%d_%s_wifiDirect.conf" % (os.getpid(), sta.name)
```

Comment:
It should be the same as before, otherwise in scenarios with WPA/WPA2 enabled we won't be able to kill the process. Or am I wrong? Please note that the process is first killed and then a new process is started.